### ISSUE TITLE
add language to user profile, use it to translate email content

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#182]
to send translated messages to users (e.g. by email), we need their language in the user profile.

after this, we can do it like shown there for all places where send_mail is invoked:

http://stackoverflow.com/questions/8683676/django-sending-localized-emails-depending-on-userprofile-language-field-data

**Comment by ThomasWaldmann** [comment]
wanted to use templates, blocked by: https://code.djangoproject.com/ticket/24111

**Comment by ThomasWaldmann** [comment]
keep this on hold until we have a minimum requirement of django 1.8, so we can use its email templating.
